src/executables/sysrepoctl.1

```diff
@@ -93,7 +93,8 @@ are being changed. \fIMOD-DATASTORE\fP can be:
 .TP
 .BR "\-m\fR,\fP \-\^\-module-plugin \fIMOD-DATASTORE\fP:\fIPLUGIN-NAME\fP"
 Set specific module datastore plugin for a module datastore, can be specified multiple
-times for different module datastores. Accepted by \fBinstall\fP op.
+times for different module datastores. If \fIPLUGIN-NAME\fP is an empty string, it is set to NULL. Accepted by
```

Comment:
... which is only allowed for running datastore. Also mention that whatever datastores are not set will use the default plugin.